src/main/config/config.c

```diff
@@ -845,6 +845,10 @@ void changePidProfile(uint8_t pidProfileIndex)
     beeperConfirmationBeeps(pidProfileIndex + 1);
 }
 
+void updateRPMLimiterExpectedThrottleLimit(void) {
+    mixerInitProfile();
```

Comment:
Don't use initialization functions in update functions. Rather explicitly update what needs to be updated in here.

Reply:
Should I do that even if the code would be a copy of some of the code in mixerInitProfile?

Reply:
Sorry, forgot to answer your question... I would say that's okay here so that there is no init function inside an update function :)